### PR TITLE
camelCased mostly water_usage and carbon_output

### PR DESCRIPTION
**Pull request by Rahul-Lal**
Didn't touch knex table details.

### CODE REVIEW

File: client/components/AddFood.jsx [hunk 1]
```diff
@@ -16,9 +16,9 @@ const categories = [
 export default class AddFood extends React.Component {
   state = {
     name: '',
-    category_id: null,
-    carbon_output: null,
-    water_usage: null,
+    categoryID: null,
```
Comment (emilyparkes) [inline client/components/AddFood.jsx]: Please change this to `categoryId` (with a lower-case `d`)

File: client/components/AddFood.jsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ export default class AddFood extends React.Component {
           </Form.Field>
           <Form.Field onChange={this.handleInputChange}
             label="Food group"
-            name="category_id"
+            name="categoryID"
```
Comment (emilyparkes) [inline client/components/AddFood.jsx]: Please change this to `categoryId` (with a lower-case `d`)

File: server/db/category.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ function getCategories (db = connection) {
 
 function getFoodsByCategory (categoryName, db = connection) {
   return db('foods')
-    .join('categories', 'foods.category_id', 'categories.id')
+    .join('categories', 'foods.categoryID', 'categories.id')
```
Comment (emilyparkes) [inline server/db/category.js]: Please change this to `foods.category_id as categoryId` (with a lower-case `d`)

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: server/db/category.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ function getCategories (db = connection) {
 
 function getFoodsByCategory (categoryName, db = connection) {
   return db('foods')
-    .join('categories', 'foods.category_id', 'categories.id')
+    .join('categories', 'foods.categoryId', 'categories.id')
```
Comment (emilyparkes) [inline server/db/category.js]: This actually needs to be used like `foods.category_id` because that is what it is named in the database table.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

**Review by emilyparkes — CHANGES_REQUESTED** [review]
Pull from dev to update and then approved :)

**Comment by bryceknz** [comment]
Closes #128

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**